.prettierrc.yml

```diff
@@ -16,3 +16,4 @@ overrides:
   - files: "*.yml"
     options:
       singleQuote: false
+      printWidth: 250
```

Comment:
What does prettier even do if you go over the width limit?  (Maybe nothing... just error).

Reply:
In most cases it would find somewhere to break the line up into multiple lines. If it can't, the behavior is language dependent. For example, I've seen in JS files that it's ok with very long strings and comments, but in Python it doesn't seem to make that exception.